# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/WebRenderAPI.h
# Commit: eac17a862c29
# Full Hash: eac17a862c29a52ae1a389c5e55c0a1cec0db7b6
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2020-01-11 09:48:29
# Regressor Bug: 1600472
# File Overlap Count: 2
# Description:
#   Bug 1600472 - Disable allowing sacrificing subpixel anti-aliasing for small screens. r=jrmuizel
#   
#   This patch only allows sacrificing subpixel anti-aliasing when the
#   screen size is larger than WUXGA, and when the force disable pref is not
#   set. In the future, we may also add disable this for high end GPUs.
# ==============================================================================

diff -r b6e5faa0693d -r eac17a862c29 gfx/webrender_bindings/WebRenderAPI.h
--- a/gfx/webrender_bindings/WebRenderAPI.h	Fri Jan 10 18:24:13 2020 +0000
+++ b/gfx/webrender_bindings/WebRenderAPI.h	Fri Jan 10 16:57:55 2020 +0000
@@ -179,6 +179,8 @@
 
   void DeleteFontInstance(wr::FontInstanceKey aKey);
 
+  void UpdateQualitySettings(bool aAllowSacrificingSubpixelAA);
+
   void Notify(wr::Checkpoint aWhen, UniquePtr<NotificationHandler> aHandler);
 
   void Clear();
@@ -308,7 +310,6 @@
   bool mUseDComp;
   bool mUseTripleBuffering;
   layers::SyncHandle mSyncHandle;
-  wr::DebugFlags mDebugFlags;
   wr::RenderRoot mRenderRoot;
 
   // We maintain alive the root api to know when to shut the render backend